src/main/flight/failsafe.c

```diff
@@ -75,6 +75,8 @@ PG_RESET_TEMPLATE(failsafeConfig_t, failsafeConfig,
     .failsafe_fw_pitch_angle = 100,     // 10 deg dive (yes, positive means dive)
     .failsafe_fw_yaw_rate = -45,        // 45 deg/s left yaw (left is negative, 8s for full turn)
     .failsafe_stick_motion_threshold = 50,
+    .failsafe_min_distance = 0,            // No minimum distance for failsafe by default  
+    .failsafe_min_distance_procedure = 1   // default minimum distance failsafe procedure is 1: auto-landing, 1: drop, 2 : RTH
```

Comment:
Please don't use magic number here. Use a value from `failsafeProcedure_e` enum instead.

Reply:
Sorry, I was just following the example set by the setter for .failsafe_procedure immediately above it. I've now changed both lines to use the enum, just like Betaflight trunk already does.

Reply:
No worries!